src/components/Inputs/Button/Button.jsx

```diff
@@ -2,16 +2,35 @@ import React from "react";
 import styled from "styled-components";
 import PropTypes from "prop-types";
 
-export const Button = ({ isInverted = true, children, ...props }) => (
-  <SButton isInverted={isInverted} {...props}>
+export const Button = ({
+  /* eslint-disable react/prop-types */
+  icon, // Styled Icon type
+  color,
+  backgroundColor,
+  borderColor,
+  children,
+  ...props
+}) => (
+  <SButton
+    color={color}
+    backgroundColor={backgroundColor}
+    borderColor={borderColor}
+    {...props}
+  >
     {children}
+    {!!icon && <Icon as={icon} />}
```

Comment:
This is fine for now but in the future we will probably be using the Design Team's svg's so ```<img src={icon} alt="..." />``` might be better for images

then it would be styled
```
const Icon = styled.img`
// your changes here
`;
```